docs/source/notebooks/moment_interaction.ipynb

```diff
@@ -282,24 +282,55 @@
+    "The limits of the moment interaction diagram and any additional analysis points can be specified by defining control points. Control points are defined as a list of (``str``, ``float``) tuples. The following types of control points are used in ``concreteproperties``:\n",
     "\n",
-    "2. ``labels`` - A list of labels to apply to the ``control_points`` for plotting purposes.\n",
-    "3. ``n_points`` - The number of neutral axis depths to compute between each control point.\n",
-    "3. ``max_comp`` - Limits the maximum compressive force in the moment interaction diagram by cutting off the top of the diagram."
+    "1. ``D`` - ratio of neutral axis depth to section depth, e.g. the (``\"D\"``, ``0.5``) control point will ensure a neutral axis at mid-depth is analysed.\n",
+    "\n",
+    "2. ``d_n`` - neutral axis depth, e.g. the (``\"d_n\"``, ``200``) control point will ensure a netural axis at 200 is analysed.\n",
+    "\n",
+    "3. ``fy`` - yield ratio of the most extreme tensile bar, e.g. the (``\"fy\"``, ``0.5``) control point will ensure a neutral axis corresponding to a strain of half the yield strain in the extreme tensile bar is analysed. The (``\"fy\"``, ``1.0``) control point is the balanced point.\n",
+    "\n",
+    "4. ``N`` - axial force, e.g. (``\"N\"``, ``3000e3``) ensures a neutral axis depth corresponding to a net axial force of 3000e3 is analysed.\n",
+    "\n",
+    "5. ``kappa_0`` - zero curvature compression, this must be at start of the list as it results in the largest compression force. Note that the second value in the tuple is not used and can be anything.<br>\n",
+    "\n",
+    "The following parameters can be used to modify the moment interaction diagram:\n",
+    "\n",
+    "1. ``limits`` - List of control points that define the start and end of the interaction diagram. List length must equal two. The default limits range from decompression ``(\"D\", 1.0)`` to zero curvature tension ``(\"d_n\", 1e-6)``.\n",
+    "\n",
+    "2. ``control_points`` - adding control points to the moment interaction diagram allows the user to ensure certain points are captured on the moment interaction diagram.  The default argument for ``control_points`` is ``[(\"kappa0\", 0.0), (\"fy\", 1.0), (\"N\", 0.0)]`` which produces control points at the pure compression point, the balanced point and the pure bending point. \n",
+    "    \n",
+    "\n",
+    "3. ``labels`` - A list of labels to apply to the ``limits`` and ``control_points`` for plotting purposes. The first two values in ``labels`` apply labels to the ``limits``, the remaining values apply labels to the ``control_points``.\n",
+    "\n",
+    "4. ``n_points`` - The number of points to compute including and between the ``limits`` of the moment interaction diagram.\n",
```

Comment:
add here that `n_points` is number of points based on evenly spaced neutral axis depths?

Reply:
Thanks, changed.